### PR TITLE
[WIP] propel deploy ci integration

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -37,60 +36,138 @@ jobs:
...
+
+          # fetch service file and check it published
+          autonomy init --reset --author ci --ipfs --remote
+          autonomy fetch $IPFS_HASH --service --alias $SERVICE_PATH
 
+          # cut of S prefix from vars
+          sed -e "s/ PRODUCTION_/ /g" -i github.vars
+          
+          # set proper CMD url
+          export CMD="propel -U https://app.propel.valory.xyz"
```
Comment (DavidMinarsch) [inline .github/workflows/release.yaml:164]: So why am I providing this as an env var if we set it here hardcoded?

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -37,60 +36,138 @@ jobs:
...
+        run: |
+          # determine ipfs hash id
+          export IPFS_HASH=$(jq '.dev | to_entries[] | select(.key | startswith("service"))| .value' -r  ./packages/packages.json | head -n1)
+          export SERVICE_PATH=service_for_propel
+
+          # fetch service file and check it published
+          autonomy init --reset --author ci --ipfs --remote
+          autonomy fetch $IPFS_HASH --service --alias $SERVICE_PATH
 
+          # cut of S prefix from vars
```
Comment (DavidMinarsch) [inline .github/workflows/release.yaml:160]: So variables are auto-prepended with the environment name?

File: .github/workflows/test_release.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+name: Release Flow
```
Comment (DavidMinarsch) [inline .github/workflows/test_release.yaml:1]: Do we still need this file here?

**Comment by solarw** [comment]
was a tmp PR. closing now

**Pull request closed without merging**